com.kms.cura.entity/src/com/kms/cura/entity/user/PatientUserEntity.java

```diff
@@ -21,7 +25,10 @@ public PatientUserEntity(String id, String name, String email, String password,
 	}
 
 	public String getGender() {
-		return gender;
+		if(gender.equals(GENDER_MALE)){
+			return "Male";
+		}
+		return "Female";
```

Comment:
@hieumai  I don't get it. The R.string. can not call outside the android project